scanpy/external/pp/_scvi.py

```diff
@@ -1,11 +1,20 @@
+import warnings
 import numpy as np
 import pandas as pd
 import scipy as sp
 
 from typing import Optional, Sequence, Union
 from anndata import AnnData
 
-MIN_VERSION = "0.6.5"
+MIN_VERSION = "0.6.7"
+
+warnings.filterwarnings("default", category=DeprecationWarning, module=__name__)
+
+warnings.warn(
+    "scvi via scanpy external API is no longer supported. "
+    + "Please use the new scvi-tools package from `scvi-tools.org`",
+    DeprecationWarning,
+)
```

Comment:
* Could this go inside the `scvi` function (ahead of the imports) instead?
* A [`FutureWarning`](https://docs.python.org/3/library/exceptions.html#FutureWarning) can be used to make deprecation warnings that are user visible
* Please try to avoid side effects like modifying the global warnings filter.